CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+# Contributing
+
+Contributions are welcome!
+
+## Code of Conduct
+
+Before contributing to this repository for the first time, please review our project's [Code of Conduct](https://github.com/devfile/api/blob/main/CODE_OF_CONDUCT.md).
+
+## Getting Started
+
+### Issues
+
+- Open or search for [issues](https://github.com/devfile/api/issues) with the label `area/api`.
+
+- If a related issue doesn't exist, you can open a new issue using a relevant [issue form](https://github.com/devfile/api/issues/new/choose). You can tag issues with `/area api`.
+
+### Building
+
+To build the CRD and the various schemas, you don't need to install any pre-requisite apart from `docker`.
+In the root directory, just run the following command:
+
+```
+./docker-run.sh ./build.sh
+```
```

Comment:
Worth noting (perhaps) that on some SELinux configurations, this command fails "permission denied". The current workaround I use is to add `:Z` to the volume mount, though this is probably not ideal.

Reply:
:+1: makes sense to me